apps/src/templates/lessonOverview/LessonStandards.jsx

```diff
@@ -3,6 +3,50 @@ import React, {PureComponent} from 'react';
 import _ from 'lodash';
 import {standardShape} from './lessonPlanShapes';
 
+export const ExpandMode = {
+  NONE: 'none',
+  FIRST: 'first',
+  ALL: 'all'
+};
+
+const expandModeShape = PropTypes.oneOf(
+  // The component should not be expanded.
+  ExpandMode.NONE,
+
+  // The component should be expanded. Its first child and the first child of
+  // each of its descendants should also be expanded.
+  ExpandMode.FIRST,
+
+  // This component and all its descendants should be expanded.
+  ExpandMode.ALL
+);
+
+/**
+ * Given the parents expand mode and the index of the child, returns what the
+ * expand mode of the child should be.
+ * @param parentExpandMode {ExpandMode} Expand mode of parent component
+ * @param index {number} index of the child with respect to parent
+ * @returns {ExpandMode} Expand mode of child component
+ */
+function getNextExpandMode(parentExpandMode, index) {
```

Comment:
nit: to me it would be clearer if this was named `getNextChildExpandMode` as it's not immediately clear to me that this function returns the child's expand mode. The comments are clear though, so totally fine if you disagree

Reply:
makes sense. I renamed to `getChildExpandMode`, since its shorter and I think "next" was really trying to mean "child". let me know if `getNextChildExpandMode` sounds better.

Reply:
That sounds good to me!